generators/app/index.js

```diff
@@ -43,8 +45,9 @@ module.exports = class extends Generator {
     this._copy('templates/imports/lib/layouts/app-layout.html')
     this._copy('templates/imports/lib/layouts/app-layout.js')
     this._copy('templates/imports/lib/layouts/preloader.html')
-
+    this._copy('templates/imports/lib/config/layout.js')
```

Comment:
this can be better named. if it's specific to body-class and it is for view only, then maybe we can put it into `/imports/views/body-class.js`